src/frontend/magnify/jest-setup.ts

```diff
@@ -1,3 +1,4 @@
-import "@testing-library/jest-dom";
+import '@testing-library/jest-dom';
 
 Object.defineProperty(window, 'scrollTo', { value: jest.fn, writable: true });
+jest.setTimeout(10000);
```

Comment:
I splited the test script of the circleci because the "runInBand" was aplied only to one of the two test scripts. Now I is much faster on circleci. On my computer, I see a net fall of performance when storybook is running too... I applied the timout only to the longest test. It should be enough event in bad circumstances now.